### ISSUE TITLE
Deployment Failure

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/risk_assessment#137]
From the `master` branch, deployment to shinyapps.io appeared to go fine from the deployment console:

![image](https://user-images.githubusercontent.com/17878953/140516792-e6234252-96e6-4af0-b720-22174b858000.png)

But when you go to the site:

![image](https://user-images.githubusercontent.com/17878953/140516857-c3288322-588b-471c-ad54-48533f1978f5.png)

Before deploying, it did find an issue with line 114 of `dbupload.R`, stating that the "Filepaths are case-sensitive on deployment server."

![image](https://user-images.githubusercontent.com/17878953/140516914-081d2609-7e05-41de-b05f-b74d91715386.png)

Note: @marlycormar said she failed to deploy as well, but received a different error when deploying the `jsm` branch.

**Comment by AARON-CLARK** [comment]
Solution was simple and actually well documented via [comments ](https://github.com/pharmaR/risk_assessment/blob/34fafe42e400c921ebf2badc8548a3f68752fd58/global.R#L69)in the `global.R` file. I created a new branch called `depoly` which successfully deployed torpharma's shinyapps.io account [here](https://rinpharma.shinyapps.io/risk_assessment_0_1_1/). Note that it had to change the name of the application slightly so that it didn't overwrite the previously deployed version BECAUSE it errored out trying to remove it. I named it risk_assessment_0_1_1 because it's designed to be paired with `riskmetric 0.1.1`.

Leaving this issue open however because we are going to push some other PRs through first before sharing the URL with others.
